pages/index.js

```diff
@@ -92,79 +93,79 @@ const Home = () => {
 
   return (
     <Layout>
-      <OONIRunHero href={'https://ooni.torproject.org'} />
+      <OONIRunHero href={'https://ooni.torproject.org'}  />
 
-      <Container pt={4} maxWidth={800}>
+      <Container pt={4} sx={{maxWidth: '800px !important'}}>
         <Flex justifyContent='center'>
           <Box width={3 / 4}>
             <Heading h={2}><FormattedMessage id='Title.WhatCanYouDo' defaultMessage='What you can do' /></Heading>
             <FormattedMessage
-              tagName={Text}
               id='WhatCanYouDoText.WebCensorship'
               defaultMessage='Generate a link and share it with your friends and contacts around the world. Encourage them to run OONI Probe to test the sites of your choice!'
-            />
+            >
+              {txt => <Text as='p' lineHeight={1.25}>{txt}</Text>}
+            </FormattedMessage>
             <URLs onSubmit={onSubmitURLs} />
           </Box>
         </Flex>
+        
+          <Modal
+            onHideClick={() => setShowModal(false)}
+            show={showModal}
+            closeButton="right"
+            width={[9 / 10, 7 / 10]}
+            sx={{borderRadius: 20}}
+          >
```

Comment:
I think it's just formatting changes.
Earlier when Modal was not working properly, I had to do some changes but when I solved a bug with `Modal` component, thing started working fine.